### DISCUSSION TITLE
Go to V

### DISCUSSION THREAD

**Discussion by ShalokShalom** [vlang/v#15771] [Ideas]
I see potential for the ecosystem and the language as a whole, to develop some sort of Go > V transpiler. 

I think this seems quite natural.

**Comment by JalonSolov** [comment]
See https://github.com/vlang/go2v

**Reply by ShalokShalom** [reply]
Oh, nice. Is there a status update?

Current capabilities? 
Thanks

**Reply by JalonSolov** [reply]
Current status is "incomplete".  It's still WIP.

Current capabilities... some things work, some don't.  There is no itemized list, other than the tests.
